controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/Maintainer.java

```diff
@@ -52,7 +52,7 @@ public Maintainer(Controller controller, Duration interval, JobControl jobContro
         this.name = name;
         this.activeSystems = Set.copyOf(activeSystems);
 
-        service = new ScheduledThreadPoolExecutor(1);
+        service = new ScheduledThreadPoolExecutor(1, r -> new Thread(r, getClass().getSimpleName() + "-worker"));
```

Comment:
Use `name()` in case maintainer has specified a custom one.

Reply:
```suggestion
        service = new ScheduledThreadPoolExecutor(1, r -> new Thread(r, name() + "-worker"));
```